packages/template-icons/src/XStateIcon.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { SVGIcon, ISVGIconProps } from './SVGIcon';
+
+export const XStateIcon: React.FC<ISVGIconProps> = ({ ...props }) => (
+  <SVGIcon {...props} viewBox="0 0 405 405">
+    <g id="Logos" stroke="none" strokeWidth="1" fill="none" fillRule="evenodd">
```

Comment:
I think if you paste or upload the file in [SVGOMG](https://jakearchibald.github.io/svgomg/) it will strip the `id`s from the svg (and do a bunch of other optimizations). This will probably make the Jest test succeed!

Reply:
Updated. Thanks @tristandubbeld